wpeverest/user-registration
# UR-1408 Fix- PHP version 8.3 combability issue

### All Submissions:






### Changes proposed in this Pull Request:



Preciously deprecated message thrown when used with PHP version 8.3
### How to test the changes in this Pull Request:

1.Select PHP version 8.3.0 install user-registration and use every functionality and features  .
3.Deprictaed message should be shown

### Types of changes:




### Other information:




### Changelog entry

> Fix- PHP version 8.3 combability issue
